docs/conf.py

```diff
@@ -69,9 +62,8 @@
     'sphinx_autodoc_typehints',
     'reno.sphinxext',
     'nbsphinx',
-    "sphinxcontrib.jquery",  # Remove when changing html_theme to qiskit_ecosystem.
+    "qiskit_sphinx_theme",
```

Comment:
This will still activate sphinxcontrib.jquery for you, and it activates other optional functionality the theme provides.